### PR TITLE
Improve install intent UX + use non transpiled ui components

### PR DESCRIPTION
**Pull request by CPatchane**
* Redirect to configuration when this is an installed konnector
* Redirect to a success modal if installed application
* Fix some wrong function call since args changing
* Fix spinner position in intents
* __Use back non transpiled cozy-ui components since it's breaking the intents styles and don't work with the transpiled AppIcon__

screenshot 2018-11-22 at 00 56 37

### CODE REVIEW

**Comment by CPatchane** [comment]
[🚩 [STO][interapp] L'installation d'une app est répétable à l'infini](https://trello.com/c/7sZ1ClGx/478-%F0%9F%9A%A9-stointerapp-linstallation-dune-app-est-r%C3%A9p%C3%A9table-%C3%A0-linfini)

**Comment by gregorylegarec** [comment]
The first commit should not be in this PR as it fixes the production. At least it should be in a independant PR.

File: src/ducks/apps/index.js [hunk 1]
```diff
@@ -465,45 +467,69 @@ function initializeRealtime() {
   }
 }
 
+async function _getInstalledInfos(app) {
+  try {
+    let installedApp = await cozy.client.fetchJSON(
```
Comment (gregorylegarec) [inline src/ducks/apps/index.js]: This information should already be in the redux store ? I don't see any reason to perform a fetch just to know if an app is installed. But maybe I am missing a point.

File: src/ducks/apps/components/InstallSuccess.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { PureComponent } from 'react'
+
+import { ModalContent, ModalHeader } from 'cozy-ui/react/Modal'
+import { translate } from 'cozy-ui/react/I18n'
+import AppIcon from 'cozy-ui/react/AppIcon'
+import { fetchIcon } from 'ducks/apps'
+
+export class InstallSuccess extends PureComponent {
```
Comment (gregorylegarec) [inline src/ducks/apps/components/InstallSuccess.jsx]: Shouldn't this file be in `src/ducks/components/intents` ? It's only used here.

File: src/ducks/components/intents/InstallAppIntent.jsx [hunk 3]
```diff
@@ -83,6 +84,9 @@ export class InstallAppIntent extends Component {
       null
     )
     const error = !!errorKey && new Error(t(errorKey))
+    const isReadyWithoutErrors = isReady && !error
+    const doConfiguration =
```
Comment (gregorylegarec) [inline src/ducks/components/intents/InstallAppIntent.jsx]: Repeated code, should be in a function imho. Btw `do` prefix is much a function name prefix. For a variable or a function returning a boolean maybe `hasToBeConfigured` or `isNeedingConfiguration` is better, even a bit longer. What do you think ?

File: src/ducks/components/intents/InstallAppIntent.jsx [hunk 4]
```diff
@@ -34,32 +35,33 @@ export class InstallAppIntent extends Component {
...
+      if (app.type === APP_TYPE.KONNECTOR && doConfiguration) {
         await compose(
           'CREATE',
           'io.cozy.accounts',
           { slug: app.slug }
         )
       }
 
-      onTerminate(nextProps.app)
+      onTerminate(this.props.app)
```
Comment (gregorylegarec) [inline src/ducks/components/intents/InstallAppIntent.jsx]: `onTerminate` should be call into `AppInstallSucces` now, as it is always displayed. That why we got a termination message too early.

File: src/ducks/apps/index.js (hunk 1 above)
Comment (CPatchane) [reply src/ducks/apps/index.js]: The intent don't fetch all apps as the main application does. So we have to fetch this information here

File: src/ducks/apps/components/InstallSuccess.jsx (hunk 2 above)
Comment (CPatchane) [reply src/ducks/apps/components/InstallSuccess.jsx]: This component is very generic and could be use to replace the success alert in the main application and there is nothing related to the intents in it

**Pull request merged**